controller/src/main/java/org/jboss/as/controller/access/constraint/SensitiveVaultExpressionConstraint.java

```diff
@@ -139,6 +141,19 @@ protected int internalCompare(AbstractConstraintFactory other) {
             // We have no preference
             return 0;
         }
+
+        @Override
+        public Constraint getRequiredConstraint(Action.ActionEffect actionEffect, JmxAction action, JmxTarget target) {
+            //TODO fix me
+            if (VaultExpressionSensitivityConfig.INSTANCE.isSensitive(actionEffect)) {
+                if (actionEffect == Action.ActionEffect.WRITE_RUNTIME || actionEffect == Action.ActionEffect.WRITE_CONFIG) {
+                    if (action.getCurrentValue() != null && isSensitiveValue(action.getCurrentValue())) {
```

Comment:
It looks like action.getCurrentValue() will always return null.

Trying to do this check is a good idea, but I think it should be commented out with a TODO until we can sort out how to cleanly deal with the value (which could be any type). That can be separate work.